TransactionProcessorACL/Startup.cs

```diff
@@ -84,6 +89,18 @@ public void ConfigureServices(IServiceCollection services)
 
         private void ConfigureMiddlewareServices(IServiceCollection services)
         {
+            services.AddHealthChecks()
+                    .AddUrlGroup(new Uri($"{ConfigurationReader.GetValue("SecurityConfiguration", "Authority")}/health"),
+                                 name: "Security Service",
+                                 httpMethod: HttpMethod.Get,
+                                 failureStatus: HealthStatus.Unhealthy,
+                                 tags: new string[] { "security", "authorisation" })
+                    .AddUrlGroup(new Uri($"{ConfigurationReader.GetValue("AppSettings", "TransactionProcessorApi")}/health"),
+                             name: "Transaction Processor Service",
+                             httpMethod: HttpMethod.Get,
+                             failureStatus: HealthStatus.Unhealthy,
+                             tags: new string[] { "application", "transactionprocessing" });
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/TransactionProcessorACL/pullRequest?prid=6237526)